vllm/model_executor/layers/quantization/utils/fp8_utils.py

```diff
@@ -2,12 +2,15 @@
 
 # Adapted from https://github.com/sgl-project/sglang/pull/2575
 import functools
+import importlib.util
 import json
 import os
 from typing import Any, Callable, Optional, Union
 
 import torch
 
+import vllm.envs as envs
+import vllm.model_executor.layers.quantization.deepgemm as deepgemm
```

Comment:
Could you move this import into the conditional in `apply_w8a8_block_fp8_linear`?